docs/langref/src/concepts/layouting.md

```diff
@@ -1,28 +1,29 @@
 # Positioning and Layout of Elements
 
-All visual elements are shown in a window. Their position is stored in the `x` and `y`
-properties as coordinates relative to their parent element. The absolute position of an element
-in a window is calculated by adding the parent's position to the element's position. If the
-parent has a grandparent element, then that one is added as well. This calculation continues until
-the top-level element is reached.
+All visual elements are shown in a window. The `x` and `y` properties store
+the elements coordinates relative to their parent element. You calculate the
+absolute position of an element in a window by adding the parent's position to
+the element's position. If the parent has a grandparent element, then that one
+is added as well. This calculation continues until the top-level element
+is reached.
 
-The size of visual elements is stored in the `width` and `height` properties.
+The `width` and `height` properties store the size of visual elements.
 
-You can create an entire graphical user interface by placing the elements in two different
-ways:
+You can create an entire graphical user interface by placing the elements in
+two ways:
 
 * Explicitly - by setting the `x`, `y`, `width`, and `height` properties.
 * Automatically - by using layout elements.
 
-Explicit placement is great for static scenes with few elements. Layouts are suitable for
-complex user interfaces, because the geometric relationship between the elements is
-expressed in dedicated layout elements. This requires less effort to maintain and helps
-to create scalable user interfaces.
+Explicit placement is great for static scenes with few elements. Layouts are
+suitable for complex user interfaces as they handle many elements at one in
+an automated way. Layouts express geometric relationship between elements in
+dedicated layout elements.
```

Comment:
I think "scalability" is a better argument for layouts than "handle many elements at one in an automated way".

```suggestion
Explicit placement is great for static scenes with few elements. Layouts are
suitable for complex user interfaces and help create scalable user interfaces. Layout elements express geometric relationships between elements.
```